db/migrate/20160209143102_add_foreign_travel_advice_subscriber_lists.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+
+class AddForeignTravelAdviceSubscriberLists < ActiveRecord::Migration
+  def change
+    csv_path = File.join(File.dirname(__FILE__), "20160209143102_add_foreign_travel_advice_subscriber_lists.csv")
+
+    CSV.foreach(csv_path) do |gov_delivery_id, country_slug|
+      SubscriberList.create!(
+        gov_delivery_id: gov_delivery_id,
+        links: {
+          countries: [country_slug]
```

Comment:
I may be missing some context? Couldn't we use the content_ids from here:
https://github.com/alphagov/travel-advice-publisher/blob/master/lib/data/countries.yml